### PR TITLE
ref: Refactors quiz trials to use the jsPsych survey plugin

### LINKED ISSUE #235: Add quiz trials with the JsPsych survey plugin
Currently in `Debrief`, `Demographics` example trial. There's multiple plugins, should add an example for each

- [ ] [survey](https://www.jspsych.org/7.3/plugins/survey/)
- [ ] [survey-html-form](https://www.jspsych.org/7.3/plugins/survey-html-form/)
- [ ] [survey-likert](https://www.jspsych.org/7.3/plugins/survey-likert/)
- [ ] [survey-multi-choice](https://www.jspsych.org/7.3/plugins/survey-multi-choice/)
- [ ] [survey-multi-select](https://www.jspsych.org/7.3/plugins/survey-multi-select/)

https://www.jspsych.org/7.3/plugins/survey/

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Another PR on the train of upgrading Honeycomb's trials to use what jsPsych ships with! This was the last trial I had left.

- Renames `quizTrials.js` as `survey.js`
- The survey trials now use jsPsych's [survey ](https://www.jspsych.org/7.3/plugins/survey/#survey) plugin
- Deletes the debrief survey trial
- Cleans up the survey text in `language.json` in the hierarchy (the text itself isn't changed)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
   -->
     <meta
       http-equiv="Content-Security-Policy"
-      content="script-src 'self'; style-src 'self' 'unsafe-inline'; font-src http: data:"
+      content="script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; font-src http: data:"
```
Comment (broarr) [inline public/index.html:19]: Why do you want this `unsafe-eval`?

**Review by broarr — APPROVED** [review]
Neat! Is this quiz something that a lot of tasks have in common?

File: public/index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
   -->
     <meta
       http-equiv="Content-Security-Policy"
-      content="script-src 'self'; style-src 'self' 'unsafe-inline'; font-src http: data:"
```
Comment (eldu) [inline public/index.html:19]: Is there an eval you need to do? We generally try to avoid it! I didn't see one in this PR

**Review by eldu — APPROVED** [review]
I took a look but i have a question on the eval !

File: public/index.html (hunk 1 above)
Comment (RobertGemmaJr) [reply public/index.html:19]: It's needed for the `survey` plugin, though I'm not sure why

File: public/index.html (hunk 2 above)
Comment (RobertGemmaJr) [reply public/index.html:19]: It's needed for the survey plugin, though I'm not sure why

**Comment by RobertGemmaJr** [comment]
> Is this quiz something that a lot of tasks have in common?

It's more of a survey for demographics and IUS stuff but yes. Most labs now link to Qualtrics but it's definitely nice to have in Honeycomb if a lab doesn't want/need that overhead

**Pull request merged**